providers/css/webauthn-admin.css

```diff
@@ -0,0 +1,166 @@
+.webauth-register {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-align: center;
+	-ms-flex-align: center;
+	align-items: center;
+}
+.webauth-register .webauthn-error {
+	margin-left: 13px;
+}
+#webauthn-keys .busy,
+.webauth-register .busy {
+	color: #7f8284;
+	pointer-events: none;
+	-webkit-transition: opacity 0.3s ease;
+	-o-transition: opacity 0.3s ease;
+	transition: opacity 0.3s ease;
+	background-size: 30px 30px;
+	background-image: -o-linear-gradient(45deg, rgba(0, 0, 0, 0.1) 25%, transparent 25%, transparent 50%, rgba(0, 0, 0, 0.1) 50%, rgba(0, 0, 0, 0.1) 75%, transparent 75%, transparent);
+	background-image: linear-gradient(45deg, rgba(0, 0, 0, 0.1) 25%, transparent 25%, transparent 50%, rgba(0, 0, 0, 0.1) 50%, rgba(0, 0, 0, 0.1) 75%, transparent 75%, transparent);
+	-webkit-animation: barberpole 0.5s linear infinite;
+	animation: barberpole 0.5s linear infinite;
+}
+@-webkit-keyframes barberpole {
+	from {
+		background-position: 0 0;
+	}
+	to {
+		background-position: 60px 30px;
+	}
+}
+@keyframes barberpole {
+	from {
+		background-position: 0 0;
+	}
+	to {
+		background-position: 60px 30px;
+	}
+}
+.webauthn-key {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-ms-flex-wrap: wrap;
+	flex-wrap: wrap;
+	-webkit-box-align: last baseline;
+	-ms-flex-align: last baseline;
+	align-items: last baseline;
+	border-top: 1px solid #ccc;
+}
+@media screen and (max-width: 600px) {
+	.webauthn-key {
+		-webkit-box-orient: vertical;
+		-webkit-box-direction: normal;
+		-ms-flex-direction: column;
+		flex-direction: column;
+		text-align: center;
+	}
+	.webauthn-key > * {
+		min-width: 100%;
+		margin-bottom: 6px;
+	}
+}
+.webauthn-key,
+.webauthn-key * {
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+}
+.webauthn-key:last-of-type {
+	border-bottom: 1px solid #ccc;
+}
+.webauthn-key .webauthn-label {
+	-webkit-box-flex: 1;
+	-ms-flex: 1;
+	flex: 1;
+	white-space: normal;
+	padding: 3px;
+}
+@media screen and (max-width: 600px) {
+	.webauthn-key .webauthn-label {
+		font-size: 1.5em;
+		padding: 6px;
+	}
+}
+.webauthn-key .webauthn-label ~ * {
+	-webkit-box-flex: 0;
+	-ms-flex: 0;
+	flex: 0;
+}
+.webauthn-key .webauthn-created,
+.webauthn-key .webauthn-used {
+	display: inline-block;
+	white-space: nowrap;
+	padding: 0 0.5em;
+}
+@media screen and (min-width: 600px) {
+	.webauthn-key .webauthn-created,
+	.webauthn-key .webauthn-used {
+		min-width: 100px;
+	}
+}
+.webauthn-key [data-tested="tested"] {
+	color: #0085ba;
+}
+.webauthn-key [data-tested="fail"] {
+	color: #dc3232;
+}
+.webauthn-key [data-tested="fail"]::before {
+	content: "";
+}
+.webauthn-key [data-tested="untested"] {
+	color: #ccc;
```

Comment:
```suggestion
	color: #ccc;
	vertical-align: bottom;
```
Align the icon better with text.

| Before | After |
| --- | --- |
| <img width="89" alt="Screenshot 2023-04-21 at 11 55 36 AM" src="https://user-images.githubusercontent.com/1657336/233529748-37680eb0-db58-4f7c-b5be-79c885877b05.png"> | <img width="97" alt="Screenshot 2023-04-21 at 11 55 12 AM" src="https://user-images.githubusercontent.com/1657336/233529699-a9088615-7a83-42f1-80a1-065f8431db93.png"> |

Reply:
Needing `vertical-align` is usually a signal to refactor torwards flexbox. We've come a long way, but browsers still render `vertical-align` differently in various cases. Life is too short for that.